appium/webdriver/extensions/android/nativekey.py

```diff
@@ -0,0 +1,1057 @@
+
+    # Key code constant: Digital Terrestrial key.
+    # Switches to digital terrestrial broadcast service.
+    TV_TERRESTRIAL_DIGITAL = 236
+
+    # Key code constant: Satellite key.
+    # Switches to digital satellite broadcast service.
+    TV_SATELLITE = 237
+
+    # Key code constant: BS key.
+    # Switches to BS digital satellite broadcasting service available in Japan.
+    TV_SATELLITE_BS = 238
+
+    # Key code constant: CS key.
+    # Switches to CS digital satellite broadcasting service available in Japan.
+    TV_SATELLITE_CS = 239
+
+    # Key code constant: BS/CS key.
+    # Toggles between BS and CS digital satellite services.
+    TV_SATELLITE_SERVICE = 240
+
+    # Key code constant: Toggle Network key.
+    # Toggles selecting broacast services.
+    TV_NETWORK = 241
+
+    # Key code constant: Antenna/Cable key.
+    # Toggles broadcast input source between antenna and cable.
+    TV_ANTENNA_CABLE = 242
+
+    # Key code constant: HDMI #1 key.
+    # Switches to HDMI input #1.
+    TV_INPUT_HDMI_1 = 243
+
+    # Key code constant: HDMI #2 key.
+    # Switches to HDMI input #2.
+    TV_INPUT_HDMI_2 = 244
+
+    # Key code constant: HDMI #3 key.
+    # Switches to HDMI input #3.
+    TV_INPUT_HDMI_3 = 245
+
+    # Key code constant: HDMI #4 key.
+    # Switches to HDMI input #4.
+    TV_INPUT_HDMI_4 = 246
+
+    # Key code constant: Composite #1 key.
+    # Switches to composite video input #1.
+    TV_INPUT_COMPOSITE_1 = 247
+
+    # Key code constant: Composite #2 key.
+    # Switches to composite video input #2.
+    TV_INPUT_COMPOSITE_2 = 248
+
+    # Key code constant: Component #1 key.
+    # Switches to component video input #1.
+    TV_INPUT_COMPONENT_1 = 249
+
+    # Key code constant: Component #2 key.
+    # Switches to component video input #2.
+    TV_INPUT_COMPONENT_2 = 250
+
+    # Key code constant: VGA #1 key.
+    # Switches to VGA (analog RGB) input #1.
+    TV_INPUT_VGA_1 = 251
+
+    # Key code constant: Audio description key.
+    # Toggles audio description off / on.
+    TV_AUDIO_DESCRIPTION = 252
+
+    # Key code constant: Audio description mixing volume up key.
+    # Louden audio description volume as compared with normal audio volume.
+    TV_AUDIO_DESCRIPTION_MIX_UP = 253
+
+    # Key code constant: Audio description mixing volume down key.
+    # Lessen audio description volume as compared with normal audio volume.
+    TV_AUDIO_DESCRIPTION_MIX_DOWN = 254
+
+    # Key code constant: Zoom mode key.
+    # Changes Zoom mode (Normal, Full, Zoom, Wide-zoom, etc.)
+    TV_ZOOM_MODE = 255
+
+    # Key code constant: Contents menu key.
+    # Goes to the title list. Corresponds to Contents Menu (0x0B) of CEC User Control
+    # Code
+    TV_CONTENTS_MENU = 256
+
+    # Key code constant: Media context menu key.
+    # Goes to the context menu of media contents. Corresponds to Media Context-sensitive
+    # Menu (0x11) of CEC User Control Code.
+    _TV_MEDIA_CONTEXT_MENU = 257
```

Comment:
`TV_MEDIA_CONTEXT_MENU` ?

Reply:
Same as above, I found this constant [here](https://github.com/appium/java-client/blob/53814cd5ac77cf26e73c89eee49649af39f9c5a5/src/main/java/io/appium/java_client/android/nativekey/AndroidKey.java#L1181).

Reply:
Same as above.